test/common/upstream/load_balancer_impl_test.cc

```diff
@@ -403,17 +415,24 @@ TEST_P(LoadBalancerBaseTest, GentleFailoverWithExtraLevels) {
   // All levels are completely down. 100% of traffic should go to P=0
   // and P=0 should be in panic mode
   updateHostSet(host_set_, 5 /* num_hosts */, 0 /* num_healthy_hosts */);
-  updateHostSet(failover_host_set_, 5 /* num_hosts */, 0 /* num_healthy_hosts */);
-  updateHostSet(tertiary_host_set_, 5 /* num_hosts */, 0 /* num_healthy_hosts */);
-  ASSERT_THAT(getLoadPercentage(), ElementsAre(100, _, _));
+  updateHostSet(failover_host_set_, 3 /* num_hosts */, 0 /* num_healthy_hosts */);
+  updateHostSet(tertiary_host_set_, 2 /* num_hosts */, 0 /* num_healthy_hosts */);
+  ASSERT_THAT(getLoadPercentage(), ElementsAre(50, 30, 20));
   ASSERT_THAT(getDegradedLoadPercentage(), ElementsAre(0, 0, 0));
-  ASSERT_THAT(getPanic(), ElementsAre(true, _, _));
+  ASSERT_THAT(getPanic(), ElementsAre(true, true, true));
 
   // Rounding errors should be picked up by the first healthy priority.
+  // This situation is called TotalPanic, as all priority levels are in panic
```

Comment:
its a bit odd that this explanation appears here when the above example is also in total panic, maybe move it up?

Reply:
Added the same explanation to all cases when TotalPanic is used.